app/presenters/hyrax/summary_presenter.rb

```diff
@@ -67,5 +67,18 @@ def other_versions_title
       end
     return final_html
     end
+
+    def related_materials     
+      final_html= ""
+      index=0
+      related_materials_title = @solr_document.related_materials_title
+      related_materials_url = @solr_document.related_materials_url
+      related_materials_title.each_with_index do |related_title,index|
+        unless related_materials_url[index.to_i].nil?
+          final_html << '<a href="'+related_materials_url[index.to_i]+'" title="'+related_title+'" target="_blank">'+related_title+'</a><br>'
```

Comment:
@js4071 just for your awarness, Ruby offers[ more than one way of constructing Stings](https://www.designcise.com/web/tutorial/how-to-use-variables-inside-a-ruby-string), I found more intuitive to work with string interpolation. Note I'm using `'` inside HTML text.
`"<a href='#{related_materials_url[index.to_i]}' title='#{related_title}' target='_blank'>#{related_title}</a><br>"`
But there is nothing wrong with your concatenation approach.